build instructions outdated?

``` bash
$ make etcd-snapshot-k8s-service
make: *** No rule to make target 'etcd-snapshot-k8s-service'.  Stop.
```

``` bash
$ make talos-backup
make[1]: Entering directory '/home/vseva/data/projects/talos-backup'
make[2]: Entering directory '/home/vseva/data/projects/talos-backup'
[+] Building 7.9s (30/30) FINISHED                                                                                                                                                                                                                                                         [...]
make[2]: Leaving directory '/home/vseva/data/projects/talos-backup'
make[1]: Leaving directory '/home/vseva/data/projects/talos-backup'
```

``` bash
make PLATFORM=linux/arm64,linux/amd64 USERNAME=linuxmaniac PUSH=true TAG=latest image-etcd-snapshot-k8s-service
make: *** No rule to make target ' image-etcd-snapshot-k8s-service'.  Stop.
```

``` bash
make PLATFORM=linux/arm64,linux/amd64 USERNAME=linuxmaniac PUSH=true TAG=latest image-talos-backup
make[1]: Entering directory '/home/vseva/data/projects/talos-backup'
[...]
```

--- a/README.md
+++ b/README.md
@@ -47,11 +47,11 @@ kubectl create job --from=cronjob/talos-backup my-test-job
 You may build the binary with:
 
 ```bash
-make etcd-snapshot-k8s-service
+make talos-backup
 ```
 
 or as a container image with:
 
 ```bash
-make REGISTRY=registry.example.com USERNAME=myusername PUSH=true TAG=latest image-etcd-snapshot-k8s-service
+make REGISTRY=registry.example.com USERNAME=myusername PUSH=true TAG=latest image-talos-backup
 ```

